prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -308,7 +290,7 @@ def get_network_adapter_data(self) -> Dict[str, int]:
             network_adapters: Optional[Dict[str, Any]] = self._verify_redfish_call(
                 self.redfish_obj, network_adapters_root_uri
             )
-            if network_adapters is None:
+            if not network_adapters:
                 logger.debug("No network adapters could be found on chassis id: %s", chassis_id)
                 print("here")
```

Comment:
**nitpick:** 

Remove print.

Reply:
I caught this one already but it shouldn't have gone into the commit in the first place. Thanks for pointing it out though!